CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+4.0.0 (2018-02-10)
```

Comment:
Not sure if this change warrants a new major version.
I think it'd make sense to use a new major version if we drop Python 2 support, but not for dropping support for old Django versions.